.github/workflows/tests.yml

```diff
@@ -5,10 +5,10 @@ name: Tests
 
 on:
   push:
-    branches: [main]
+    branches: [ui-svelte/main]
 
-  pull_request_target:
-    branches: [main]
+  pull_request:
```

Comment:
I changed this to `pull_request` so that we can have faster feedback loop on adding svelte e2e tests.

(`pull_request_target` always runs tests against `main`, which can be a pain if you're modifying `tests.yml` on a different branch)